cat/CaT-service.yaml

```diff
@@ -2466,17 +2466,21 @@ paths:
         - OAuth2:
             - buyer
       summary: Supplier Complete
-  /tenders/projects/{proc-id}/events/{event-id}/responses/furtherInfo:
+      
+  /tenders/projects/{proc-id}/events/{event-id}/messages:
     parameters:
       - $ref: '#/components/parameters/ProcurementParam'
       - $ref: '#/components/parameters/EventParam'
-
+      - $ref: '#/components/parameters/MessageDirectionParam'
+      - $ref: '#/components/parameters/MessageSortParam'
+      - $ref: '#/components/parameters/PageParam'
+      - $ref: '#/components/parameters/PageSizeParam'
```

Comment:
These last 4 are also showing up as params for the POST operation - think you can move them to just appear under the `get:` section

Reply:
Made fix in latest version, thanks